frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.html

```diff
@@ -3,14 +3,17 @@
 <p>{{ text.confirmation_info(created, email) }}</p>
 <p>
   <span>{{ text.status_label }} </span>
-  <span *ngIf="!eeTrialService.confirmed; else confirmedStatus" class="status--waiting">
+  <span *ngIf="!eeTrialService.confirmed; else confirmedStatus" class="op-ee-trial-waiting-status--waiting"
+  data-qa-selector="op-ee-trial-waiting-status--waiting">
     {{ text.status_waiting }}
 
-    <a id="resend-link" (click)="resendMail()">{{ text.resend }}</a>
+    <a id="op-ee-trial-waiting-resend-link"
+    data-qa-selector='op-ee-trial-waiting-resend-link' (click)="resendMail()">{{ text.resend }}</a>
```

Comment:
Can you make this a `button type="button"` while we're at it? And if possible, remove the ID and only use classes

Reply:
Also, if this should just look like a basic link, then I recommend `op-link`, which also works really well on buttons ;)